types/index.d.ts

```diff
@@ -90,6 +90,25 @@ export function render(
   options?: Omit<RenderOptions, 'queries'>,
 ): RenderResult
 
+// TODO JSDOC
+interface RenderHookResult<Result, Props> {
+  rerender: (props?: Props) => void
+  result: {current: Result}
+  unmount: () => void
+}
+
+// TODO JSDOC
+interface RenderHookOptions<Props> {
+  initialProps?: Props
+  wrapper?: React.ComponentType
+}
+
+// TODO JSDOC
+export function renderHook<Result, Props>(
+  render: (initialProps: Props) => Result,
+  options?: RenderHookOptions<Props>,
+): RenderHookResult<Result, Props>
+
```

Comment:
when `options` or `options.initialProps` is not defined, `initialProps` of `render` will be undefined.

```tsx
interface RenderHookResult<Result, Props> {
  rerender: (props: Props) => void;
  result: { current: Result };
  unmount: () => void;
}

interface RenderHookOptions<Props> {
  initialProps: Props;
  wrapper?: ComponentType;
  strict?: boolean;
}

interface RenderHookResultWithoutProps<Result, Props> {
  rerender: () => void;
  result: { current: Result };
  unmount: () => void;
}

interface RenderHookOptionsWithoutProps<Props> {
  wrapper?: ComponentType;
  strict?: boolean;
}

export function renderHook<Result, Props>(
  renderCallback: () => Result,
  options?: RenderHookOptionsWithoutProps<Props>,
): RenderHookResultWithoutProps<Result, Props>;
export function renderHook<Result, Props>(
  renderCallback: (props: Props) => Result,
  options: RenderHookOptions<Props>,
): RenderHookResult<Result, Props>;
export function renderHook<Result, Props>(
  renderCallback: (props?: Props) => Result,
  options?: Partial<RenderHookOptions<Props>>,
):
```

Reply:
I think this is fine and the types are correct. Is that not the case?